src/vivarium_csu_alzheimers/data/loader.py

```diff
@@ -240,3 +245,128 @@ def reshape_to_vivarium_format(df, location):
+        scnd_group_future_year = tuple(
+            map(add, list(index), ["", "", 2 * W, 2 * W, DUR, DUR])
+        )
+        # cap age and year
+        age_end = min(100, next_group_future_year[3])
+        next_group_future_year = next_group_future_year[:2] + (
+            min(95, next_group_future_year[2]),
+            age_end if age_end < 100 else 125,
+            min(2050, next_group_future_year[4]),
+            min(2051, next_group_future_year[5]),
+        )
+        age_end = min(100, scnd_group_future_year[3])
+        scnd_group_future_year = scnd_group_future_year[:2] + (
+            min(95, scnd_group_future_year[2]),
+            age_end if age_end < 100 else 125,
+            min(2050, scnd_group_future_year[4]),
+            min(2051, scnd_group_future_year[5]),
+        )
+        # get age group incidences
+        # TBD should new_bbbm_people use inc draws? currently single value
+        next_group_inc = inc.loc[next_group_future_year[1:4] + (2021, 2022)].mean()
+        scnd_group_inc = inc.loc[scnd_group_future_year[1:4] + (2021, 2022)].mean()
+
+        # set value
+        # pdb.set_trace()
+        # print([next_group_inc, pop.loc[next_group_future_year].value, scnd_group_inc, pop.loc[scnd_group_future_year].value])
+        new_bbbm_people.loc[index] = (
+            (1 - (R / W)) * next_group_inc * pop.loc[next_group_future_year].value
+        ) + ((R / W) * scnd_group_inc * pop.loc[scnd_group_future_year].value)
+    # TBD add deaths and scale
+    # pdb.set_trace()
+    return new_bbbm_people
+
+
+def load_mci_disability_weight(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> pd.DataFrame:
+    data = pd.read_csv("/ihme/epi/disability_weights/standard/dw_full.csv")
+    motor_data = (
+        data[data.healthstate.notnull() & data.healthstate.str.contains("motor")]
+        .set_index("healthstate")
+        .filter(like="draw")
+    )
+    motor_mild = motor_data.loc["motor_mild"]
+    motor_cog_mild = motor_data.loc["motor_cog_mild"]
+    df_dw_mci = (motor_cog_mild - motor_mild) / (1 - motor_mild)
+
+    # use demography to cast draws to all ages/sexes
+    demography = get_data(data_keys.POPULATION.DEMOGRAPHY, location)
+    data = pd.DataFrame([df_dw_mci], index=demography.index)
+    data.index = data.index.droplevel("location")
+    return data
+
+
+def load_ad_dementia_state_incidence_rate(
+    key: str, location: str, years: int | str | list[int] | None = None
+) -> pd.DataFrame:
+    acmr = get_data(data_keys.POPULATION.ACMR, location, years)
+    csmr = get_data(data_keys.ALZHEIMERS.CSMR, location, years).droplevel(
+        ["year_start", "year_end"]
+    )
+
+    # for now, assume csmr is the same for all years based on docs
+    csmr_all_years = pd.DataFrame(csmr, index=acmr.index)
+    mort_MCI = acmr - csmr_all_years  # note that emr_MCI is 0
+    return (1 / data_values.MCI_AVG_DURATION) - mort_MCI
```

Comment:
FIXME: this results in some negative values for oldest age groups, so we expect the sim will fail to validate due to missing incidence count targets for these old age simulants